sdk/communication/azure-communication-chat/azure/communication/chat/_shared/models.py

```diff
@@ -22,7 +22,6 @@ class CommunicationIdentifierKind(str, Enum, metaclass=CaseInsensitiveEnumMeta):
     COMMUNICATION_USER = "communication_user"
     PHONE_NUMBER = "phone_number"
     MICROSOFT_TEAMS_USER = "microsoft_teams_user"
-    MICROSOFT_BOT = "microsoft_bot"
```

Comment:
@annatisch I tried to deprecate this enum for rooms & callautomation with using __getatribute__, __getattr_ method of the enum class but it did not work. Can you please guide me what is the proper way deprecate enum values?

Reply:
I think `getattr` and `getitem` are defined on `CaseInsensitiveEnumMeta`, so you would need to inherit from this to define a new metaclass.